service/src/main/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleService.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import com.launchdarkly.client.LDClient;
+import com.launchdarkly.client.LDUser;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+public class FeatureToggleService {
+
+    private final LDClient ldClient;
+    private final String ldUserKey;
+
+    @Autowired
+    public FeatureToggleService(LDClient ldClient, @Value("${ld.user_key}") String ldUserKey) {
+        this.ldClient = ldClient;
+        this.ldUserKey = ldUserKey;
+    }
+
+    public boolean isXeroxPrintingEnabled() {
+        return ldClient.boolVariation("xerox-printing", getUser(), false);
+    }
+
+    private LDUser getUser() {
+        return new LDUser.Builder(ldUserKey)
+            .custom("timestamp", String.valueOf(System.currentTimeMillis()))
+            .build();
+    }
```

Comment:
LDClient is thread safe, thus can be created once in constructor, instead each time when toggle is checked